src/Raven.Server/ServerWide/Commands/ClusterTransactionCommand.cs

```diff
@@ -69,6 +76,7 @@ public class ClusterTransactionOptions : IDynamicJson
             public TimeSpan? WaitForIndexesTimeout;
             public bool WaitForIndexThrow;
             public List<string> SpecifiedIndexesQueryString;
+            public bool DisableAtomicDocumentWrites;
```

Comment:
Double-check. For old commands (e.g. from 4.2 era) it will be set to false when Cluster Log is applied. We want that?